webapp/src/js/components/containers/FindGene.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import ImmutablePropTypes from 'react-immutable-proptypes';
+import _isFunction from 'lodash/isFunction';
+
+
+// Mixins
+import PureRenderMixin from 'mixins/PureRenderMixin';
+import FluxMixin from 'mixins/FluxMixin';
+import ConfigMixin from 'mixins/ConfigMixin';
+
+// UI components
+import TabbedArea from 'ui/TabbedArea';
+import TabPane from 'ui/TabPane';
+
+// Containers
+import FindGeneByNameDesc from 'containers/FindGeneByNameDesc';
+import FindGeneByRegion from 'containers/FindGeneByRegion';
+import RecentlyFoundGenes from 'containers/RecentlyFoundGenes';
+
+let FindGene = React.createClass({
+  mixins: [
+    PureRenderMixin,
+    FluxMixin,
+    ConfigMixin
+  ],
+
+  propTypes: {
+    componentUpdate: React.PropTypes.func.isRequired,
+    activeTab: React.PropTypes.string
+  },
+
+  getDefaultProps: function() {
+    return {
+      activeTab: 'tab_0'
+    };
+  },
+
+  icon() {
+    return 'bitmap:genomebrowser.png';
+  },
+
+  title() {
+    return 'Find gene';
+  },
+
+  render() {
+    let {componentUpdate, activeTab} = this.props;
+    return (
+      <TabbedArea activeTab={activeTab}
```

Comment:
An idea for simplifying this UI - no tabs. Instead show recently found if search empty, allow search to support region via strings such as 2L:0-10000 (code for doing this is at https://github.com/cggh/panoptes/blob/master/webapp/src/js/components/panoptes/genome/Controls.js#L69 which could be extracted to a module) if no valid region string then use string as normal text search.